### ISSUE TITLE
Activity Browser Tutorial

### ISSUE THREAD

**Issue by haasad** [LCA-ActivityBrowser/activity-browser#158]
Now that it appears that more people are picking up the AB, we should think about some kind of tutorial to demonstrate the basic functionalities of the AB. Not everybody will get an introduction from a developer anymore and some things are not super intuitive (eg. dragging activities to the LCA setup tab and clicking the flows in the sankey to make them expand, see questions in #154 and comment [here](https://github.com/LCA-ActivityBrowser/activity-browser/issues/152#issuecomment-387410326)). So far users were also usually familiar with brightway, which might change in the future.

I could imagine several options:
- tutorial video
- simple markdown file in the repo
- use the wiki here on github

Video is probably more of a long-term solution, but I feel we should think about adding something basic rather soon. I'd personally go with the Wiki for now. Any preferences/ideas @bsteubing? Again ping @TMillross, might be nice to have for the conference.

**Comment by TMillross** [comment]
I agree with the wiki idea for offering the best result for not too much effort. 
A video only makes sense once there's some stability in the interface I think. Things are still changing at quite a rate, so it would become outdated quickly.

**Comment by TMillross** [comment]
Ok great. Makes sense to use the standard GitHub wiki right? Just tested on my fork and [enabling it is trivial](https://help.github.com/articles/adding-wiki-pages-via-the-online-interface/). @haasad happy to go for this? Or anyone else if you have sufficient access rights :)

**Comment by bsteubing** [comment]
@cardosan : Nice to hear about your course and the idea of using BW / AB in teaching. We have similar ideas here in Leiden, but to be honest, I think we need still to invest a bit more to make the AB good enough for teaching. Actually, ETH was also thinking about this, so if we could pool our resources, perhaps we could tap some funding somewhere to make this happen. Let's perhaps stick our heads together on this issue at some point this summer.

**Comment by cardosan** [comment]
@bsteubing based also on what  @TMillross and @pjamesjoyce said in #153 and #149
>  On the teaching front, we're developing a couple of MSc level courses in collaboration with the biotech and chemical engineering departments here at KTH on, essentially, LCA for non-LCA people, using lcopt for the practicals instead of SimaPro - I'll report back on how they go 

> From presenting the AB poster at an IE conference last month, we can testify to this as a widespread desire! A number of professors stated an intention to try the AB & BW combo for teaching in the coming years

> A number of professors teaching LCA at Bachelors, Masters or Professional level were present. They represented Santa Barbara, Toronto & Michigan (and some others who's affiliations I did not record). They were broadly keen to use AB for their courses in the coming years. 

it seems that the interest in having an open source platform for teaching LCA is quite widespread. 
I would strongly support the idea of trying to team up to continue a more harmonized development of all the bw2-based tools (but any OS LCA is obviously welcome!) and develop some sort of teaching platform for OS LCA....maybe something similar to what the industrial ecologist [are doing](http://www.teaching.industrialecology.uni-freiburg.de/) Why not thinking also to a MOOC in in the longer term?

**Comment by TMillross** [comment]
@cardosan the IE MOOC you link to (written by @stefanpauliuk) includes a section "Advanced LCA exercises with openLCA. An ecoinvent license is required". If we develop high quality teaching materials with BW&AB then I suspect Stefan would also be happy to link to our work near this section. 

My experience with learning BW through the [Jupyter notebooks](https://docs.brightwaylca.org/notebooks.html) was positive. There are also [these notebooks](https://github.com/PoutineAndRosti/Brightway-Seminar-2017) from a seminar last year, which I haven't tried. Activity Browser is not likely to ever handle all the possibilities of BW, so to reach the highest capabilities, Python skills will still be essential for the foreseeable. But our tool will soon meet the needs of many use cases including those of those new to LCA, so any tutorials could (should?) start at a beginner level.

I'd further argue that we should only use open (free and easily available) databases for any educational materials developed, so it's easy for everyone to follow, regardless of licensing.

**Comment by cardosan** [comment]
@tmillross any news about the lcopt course you are preparing?
I am also gonna use lcopt for mine and would be happy to give a look to what you have done.

P.S: I am still in for the idea of @bsteubing of pooling resources to make a some joint teaching material, what do you think, are the time mature to think concretely about this?

**Comment by tmillross** [comment]
@cardosan Are you referring to [this comment](https://github.com/LCA-ActivityBrowser/activity-browser/issues/153#issuecomment-398088092)? I was tagged, but not the one who wrote it :D

**Comment by marc-vdm** [comment]
Move [tutorial](https://github.com/LCA-ActivityBrowser/activity-browser/wiki/Tutorials#your-first-lca)  to new io site for 3.0
